packages/components/translations/en-us.yaml

```
@@ -0,0 +1,44 @@
+hds:
+  components:
+    shared:
```

maybe? I have mixed feelings... 

```suggestion
    common:
```

---

I'm fine with `common`